src/lib/renderer/renderer.tsx

```diff
@@ -65,22 +73,59 @@ export interface IRenderFormProps {
  * All components receive the `IComponentProps` as props containing the required context to render
  * the component. Components should return a React.ReactElement.
  *
- * @external {CallbacksContext} Expects `CallbackContext` to be available.
- * @external {RenderContext} Expects `RenderContext` to be available.
- * @external {SubmissionContext} Expects `SubmissionContext` to be available.
+ * @external {RenderContext} Provides `RenderContext`.
+ * @external {CallbacksContext} Provides `CallbackContext`.
+ * @external {SubmissionContext} Provides `SubmissionContext`.
  */
-export const RenderForm = ({form}: IRenderFormProps): React.ReactElement => {
-  const children =
+export const RenderForm = ({
+  children,
+  configuration,
+  form,
+  formAttrs,
+  initialValues,
```

Comment:
this doesn't make a lot of sense since `initialValues` is in essence equal to your `submission.data` - given an existing submission with data being loaded (or default values), that is the starting point for formik. There is no situation where you provide a submission with data AND initial values that are different.